sdk/agrifood/azure-verticals-agrifood-farming/pom.xml

```diff
@@ -32,6 +32,9 @@
   <properties>
     <!-- Anomaly detector temporarily skipping code coverage until tests are added -->
     <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
+    <javaModulesSurefireArgLine>
+      --add-exports com.azure.core/com.azure.core.implementation.jackson=ALL-UNNAMED
```

Comment:
What caused this to be needed? Was this a problem with getting an `ObjectMapperShim` from `azure-core` due to the lack of exporting of `implementation`? If so, I'd love to look into adding a sample for SDK libraries on using `ObjectMapperShim` which can help flush out any potential Java 9+ module scopes.

Reply:
So the issue here is somewhere between junit5 and surefire and seems like a bug. Junit test module has to have access to `implementation.jackson`. Some context here https://github.com/junit-team/junit5/issues/2147

`Implementation.jackon` package is exported to `management` and `json-jackson` modules, so it's not available to junit  module created. We have a similar configuration for other packages exported to a limited set of internal packages.

This has no effect on jar built - end-users don't need access to `ObjectMapperShim` and serialization works.